src/ripple/overlay/impl/Manifest.cpp

```diff
@@ -206,6 +205,25 @@ ManifestCache::configManifest (
     }
 }
 
+boost::optional<std::string>
+ManifestCache::member (PublicKey const& identity) const
```

Comment:
Returning the `comment` as an optional seems strange here. They way `member` is used is to check if if the key is part of the map. Why not just return a `bool`?